CHANGELOG-2.0.md

```diff
@@ -72,12 +72,12 @@ We've replaced the default measure-tools provided in the Viewer from itwinjs-cor
 
 - A [codemod](https://github.com/iTwin/codemods) has been developed to help you upgrade your exisitng code from iModel.js@2.x to iTwin.js@3.0.
 
-- Change the version of `@itwin/web-viewer-react` in your package.json to use the `"latest"` dist tag, eg:
+- Change the version of `@itwin/web-viewer-react` in your package.json to the following:
 
   ```
     "dependencies": {
       ...
-      "@itwin/web-viewer-react": "latest",
+      "@itwin/web-viewer-react": "2",
```

Comment:
should this be ^2.0.0? or is this some shorthand that I am not aware of

Reply:
im pretty sure this was a way to specify ^2.x, but ill be more explicit